libraries/AP_ExternalAHRS/AP_ExternalAHRS_InertialLabs.cpp

```diff
@@ -62,11 +62,11 @@ extern const AP_HAL::HAL &hal;
 #define ILABS_UNIT_STATUS2_BARO_FAIL              0x0008
 #define ILABS_UNIT_STATUS2_DIFF_PRESS_FAIL        0x0010
 #define ILABS_UNIT_STATUS2_MAGCAL_2D_ACT          0x0020
-#define ILABS_UNIT_STATUS2_MAGCAL_3D_ACT          0x0020
-#define ILABS_UNIT_STATUS2_GNSS_FUSION_OFF        0x0040
-#define ILABS_UNIT_STATUS2_DIFF_PRESS_FUSION_OFF  0x0080
-#define ILABS_UNIT_STATUS2_MAG_FUSION_OFF         0x0100
-#define ILABS_UNIT_STATUS2_GNSS_POS_VALID         0x0200
+#define ILABS_UNIT_STATUS2_MAGCAL_3D_ACT          0x0040
```

Comment:
So this looks bad - do you think this was a transcription error from the original author?

Reply:
As in the original code is wrong, and the datasheet specifies these as you've put them in now?

Reply:
For convenience I can attach the tech-manual of this EAHRS
Right version of these flags mentioned on page 222 (USW2 structure table)

[INS_ICD_rev-2.80_Jan_2024.pdf](https://github.com/user-attachments/files/16691420/INS_ICD_rev-2.80_Jan_2024.pdf)